bridgev2/matrix/intent.go

```diff
@@ -674,3 +674,7 @@ func (as *ASIntent) MuteRoom(ctx context.Context, roomID id.RoomID, until time.T
 		})
 	}
 }
+
+func (as *ASIntent) GetEvent(ctx context.Context, roomID id.RoomID, eventID id.EventID) (*event.Event, error) {
+	return as.Matrix.GetEvent(ctx, roomID, eventID)
```

Comment:
This is missing decryption and the content also need to be parsed

1. After getting event, if there's no error, call `evt.Content.ParseRaw(evt.Type)`. The error from that can be logged, but no need to return it
2. If `evt.Type` is `event.EventEncrypted`, you need to decrypt it. However, if `Bridge.Config.Encryption.DeleteKeys.RatchetOnDecrypt` is true, don't even try decrypting, just return an error. After successfully decrypting, call ParseRaw again on the result before returning it